python/foglamp/services/south/server.py

```diff
@@ -215,14 +186,40 @@ def run(self):
         asyncio.ensure_future(self._start(loop))
         loop.run_forever()
 
+    async def _stop(self, loop):
+        if self._plugin is not None:
+            try:
+                self._plugin.plugin_shutdown(self._plugin_handle)
+            except Exception:
+                _LOGGER.exception("Unable to stop plugin '{}'".format(self._name))
+            finally:
+                self._plugin = None
+                self._plugin_handle = None
+
+        try:
+            await Ingest.stop()
+        except Exception:
+            _LOGGER.exception('Unable to stop the Ingest server')
+            return
+
+        # Finish all pending asyncio tasks
+        pending = asyncio.Task.all_tasks()
+        asyncio.gather(*pending)
+
+        loop.stop()
```

Comment:
please add comments describing that this will ensure shutdown is done when response is sent back (i.e. aiohttp lib shall autohandle graceful shutdown)